variables.tf

```
@@ -220,7 +220,7 @@ variable "node_security_group_description" {
 }
 
 variable "node_security_group_additional_rules" {
-  description = "List of additional security group rules to add to the node security group created"
+  description = "List of additional security group rules to add to the node security group created. You may use `source_cluster_security_group = true` inside the rule to set the `cluster_security_group` as source"
```

```suggestion
  description = "List of additional security group rules to add to the node security group created. Set `source_cluster_security_group = true` inside rules to set the `cluster_security_group` as source"
```

We should also change `cluster_security_group_additional_rules` description as well to match since it supports the same for node security groups:

```hcl
  description = "List of additional security group rules to add to the cluster security group created. Set `source_node_security_group = true` inside rules to set the `node_security_group` as source"
```

We'll need to update the PR title to something like `docs: Update security group additional rules description for sourcing created security groups`

Lastly, we'll need to update the docs by running `pre-commit run -a`